---
geometry_msgs/PoseStamped[] checkpoints
uint32 openclose
